metasynth/distribution/faker.py

```diff
@@ -59,3 +59,8 @@ def _param_schema(cls):
             "faker_type": {"type": "string"},
             "locale": {"type": "string"},
         }
+
+
+@metadist(implements="core.unique_faker", var_type="string")
```

Comment:
Would it also work if we just do 
`@metadist(implements="core.unique_faker", var_type="string", is_unique="true")`
without subclassing `UniqueMixin`?

Would it make sense to call `UniqueMixin` something more descriptive about what method it overloads? Such as `UniqueSampler`, `UniqueDraws`, or even `UniqueRejectionSampler`?

Reply:
The term "Mixin" is the kind of class this is, so I think that will help people familiar with the concept (or can find it on google). We can also call it `UniqueDistributionMixin` or `UniqueSamplerMixin`, etc. That might be the most descriptive. It's basically a copy from the `UniqueRegexDistribution`, but I don't want to mess with that right now, while I'm messing with the regex distributions...

It is probably possible to implement a decorator like that:

https://stackoverflow.com/questions/38405619/how-to-mixin-behavior-using-class-decorators-in-python

But honestly, I think that will just drive up the complexity a little too much..

Reply:
Agreed with everything. Either of those names you suggested are good!